src/components/Settings/Rounding.js

```diff
@@ -0,0 +1,111 @@
+// @flow
+import React, { Component, Fragment } from 'react';
+import { connect } from 'react-redux';
+import { Header } from 'semantic-ui-react';
+
+import { setRounding, setRoundingDown } from '../../actions/settings';
+import { valueFromEventTarget } from '../../core/dom';
+import Combo from './Combo';
+
+type RoundingProps = {
+  rounding: string;
+  roundingDown: string;
+  setRoundingProp: (value: any) => void;
+  setRoundingDownProp: (value: any) => void;
+}
+
+class Rounding extends Component<RoundingProps> {
+  constructor(props) {
+    super(props);
+    this.onChangeRoundingDown = this.onChangeRoundingDown.bind(this);
+    this.onChangeRounding = this.onChangeRounding.bind(this);
+    this.onRoundingDownDropDownChange = this.onRoundingDownDropDownChange.bind(this);
+    this.onRoundingDropDownChange = this.onRoundingDropDownChange.bind(this);
+  }
+
+  onRoundingDownDropDownChange: (e: Event, data: any) => void;
```

Comment:
It's better to define what `data` is in this case. `{ value: string, label: string }`